src/wix/WixToolset.Core.ExtensionCache/ExtensionCacheManager.cs

```diff
@@ -149,6 +152,18 @@ private string GetCacheFolder(bool global)
             return cacheLocation.Path;
         }
 
+        private IEnumerable<string> GetCacheFolders(bool global)
+        {
+            if (this.cacheLocations == null)
+            {
+                this.cacheLocations = this.ExtensionManager.GetCacheLocations();
+            }
+
+            var cacheLocations = this.cacheLocations.Where(l => global || l.Scope == ExtensionCacheLocationScope.Project).OrderBy(l => l.Scope).Select(l => l.Path);
```

Comment:
@barnson I've debated making the `.Where()` an `XOR` instead of `OR`. Right now (with the `OR`) the code shows both global _and_ project-level extensions if scanning globally. In other words, you see all available extensions.

The alternative is to say when looking global do NOT include project-level extensions (switch to an `XOR`). Thoughts?

Reply:
`dotnet tool list` says `List tools installed globally or locally.` so inclusive is consistent with that.